core/block_validator_test.go

```diff
@@ -35,7 +35,7 @@ func testChainConfig() *ChainConfig {
 		Forks: []*Fork{
 			&Fork{
 				Name:  "Homestead",
-				Block: big.NewInt(1150000),
+				Block: big.NewInt(0),
```

Comment:
this was here before gravity's changes, all tests are written for this, I've just reverted it back (see linked commits). and there no reason to create test chain of 1150000 block for each test, just because we need only last few blocks to test